packages/slate/src/interfaces/element.js

```diff
@@ -474,11 +584,28 @@ class ElementInterface {
 
   getDescendant(path) {
     path = this.resolvePath(path)
-    if (!path) return null
+    return this.getDescendantByPath(path)
+  }
 
-    const deep = path.flatMap(x => ['nodes', x])
-    const ret = this.getIn(deep)
-    return ret
+  /**
+   * Get a descendant node by its path.
+   *
+   * @param {List} path
+   * @return {Node|Null}
+   */
+
+  getDescendantByPath(path) {
```

Comment:
Same here, I want to avoid these `*ByPath` methods. We used to have all of our methods separated by path and key and I'd like to move us towards only supporting paths at all.